src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutPersisterReceiver.cs

```diff
@@ -19,6 +19,16 @@ class TimeoutPersisterReceiver : IDisposable
         public CriticalError CriticalError { get; set; }
         public Address DispatcherAddress { get; set; }
         public TimeSpan TimeToWaitBeforeTriggeringCriticalError { get; set; }
+        internal DateTime NextRetrieval { get; private set; } = DateTime.UtcNow;
```

Comment:
I'm not sure that that it being set to slightly earlier value makes a difference.